desktop-widgets/tab-widgets/TabDiveInformation.cpp

```diff
@@ -74,6 +79,18 @@ void TabDiveInformation::updateProfile()
 			" ", current_dive->dc.divemode == FREEDIVE));
 
 	ui->sacText->setText( mean[0] ? SACs : QString());
+
+	if (current_dive->surface_pressure.mbar == 0)
+		ui->atmPressVal->clear();			// If no atm pressure for dive then clear text box
+	else {
```

Comment:
again, coding style. If one side of an `else` requires braces, we force braces on both sides.

Reply:
this one is still there